### PR TITLE
Feature/prod 2260 arrayuse

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: docker-build-publish/action.yml [hunk 1]
```diff
@@ -33,11 +33,19 @@ runs:
...
+        
+        for index in "${!array[@]}"
+        do
+            tagsvalue+=" -t ${{ inputs.registry-login-server }}/${{ inputs.target-image-name }}:${array[index]}"
+        done
+
+        echo $tagsvalue
         cp ~/.npmrc `pwd`/docker/.npmrc
-        docker build -t ${{ inputs.registry-login-server }}/${{ inputs.target-image-name }}:${{ inputs.version }} -t ${{ inputs.registry-login-server }}/${{ inputs.target-image-name }}:latest ./docker
+        docker build $tagsvalue ./docker
```
Comment (agorina) [inline docker-build-publish/action.yml:46]: @SergiiPo improved my draft PR and now we have a true solution. Thank you!

Comment (will-faro) [reply docker-build-publish/action.yml:46]: Nice bash-fu!

**Review by will-faro — APPROVED** [review]

File: docker-build-publish/action.yml [hunk 2]
```diff
@@ -33,11 +33,19 @@ runs:
...
       run: |
+        tagsvalue=''
+        IFS=', ' read -r -a array <<< ${{ inputs.versions }}
+        
+        for index in "${!array[@]}"
+        do
+            tagsvalue+=" -t ${{ inputs.registry-login-server }}/${{ inputs.target-image-name }}:${array[index]}"
+        done
+
+        echo $tagsvalue
```
Comment (will-faro) [inline docker-build-publish/action.yml:44]: Nit: is this capture in the logs as the execution of docker below? If so, maybe remove the echo

Comment (SergiiPo) [reply docker-build-publish/action.yml]: Removed

**Pull request merged**